<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomTypesWindow</class>
 <widget class="QMainWindow" name="AtomTypesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atom types</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="atomTypes_layout_output">
      <item>
       <widget class="QLabel" name="atomTypes_label_output">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Output file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="atomTypes_lineEdit_output">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Path to output file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_output">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="atomTypes_layout_mol">
      <item>
       <widget class="QLabel" name="atomTypes_label_mol">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Molecule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="atomTypes_comboBox_mol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of molecule</string>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add new molecule</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="atomTypes_label_numOfMol">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Number of molecules</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="atomTypes_spinBox_numOfMol">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of selected molecules</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="atomTypes_layout_atom">
      <item>
       <widget class="QLabel" name="atomTypes_label_chType">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Char atom type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="atomTypes_lineEdit_chType">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Character atop type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="atomTypes_label_intType">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Int atom type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="atomTypes_spinBox_intType">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Numerical atom type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_add">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add atom</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="atomTypes_listWidget">
      <property name="toolTip">
       <string>Added atoms</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="atomTypes_layout_intRem">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_typeRem">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove selected atom</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="atomTypes_layout_buttons">
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_clear">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clear window</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_create">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Create file</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="atomTypes_pushButton_close">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Close window</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>atomTypes_lineEdit_output</tabstop>
  <tabstop>atomTypes_pushButton_output</tabstop>
  <tabstop>atomTypes_comboBox_mol</tabstop>
  <tabstop>atomTypes_spinBox_numOfMol</tabstop>
  <tabstop>atomTypes_lineEdit_chType</tabstop>
  <tabstop>atomTypes_spinBox_intType</tabstop>
  <tabstop>atomTypes_pushButton_add</tabstop>
  <tabstop>atomTypes_listWidget</tabstop>
  <tabstop>atomTypes_pushButton_typeRem</tabstop>
  <tabstop>atomTypes_pushButton_clear</tabstop>
  <tabstop>atomTypes_pushButton_create</tabstop>
  <tabstop>atomTypes_pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>atomTypes_pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AtomTypesWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
